custom-words.txt

```diff
@@ -304,9 +309,11 @@ changedSince
 changepoint
 changestate
 CHECKACCESS
+Checkfeature
+CheckfeatureSupport
```

Comment:
I don't think this one is needed since "Checkfeature" is already in, and "Support" would be treated as a separate word because it is camelCase.
It's possible "Checkfeature" should be moved to a file level suppression and fixed in the spec though

Reply:
Thanks @nschonni for the reply. Can you run the spell check locally for all the specs and check the result?

Reply:
I did that over in https://github.com/Azure/azure-rest-api-specs/pull/11474, but due to the size I don't think it's going to land till smaller pieces can be landed to shrink it